sway-lib-std/src/ufp64.sw

```diff
@@ -0,0 +1,247 @@
+library ufp64;
+//! A wrapper around u64 type for a library for Sway for mathematical functions operating with signed 32.32-bit fixed point numbers. 
+
+use core::num::*;
+use ::assert::assert;
+use ::math::*;
```

Comment:
Same comment on explicit imports

Reply:
In this particular case this importing is needed to mare sqrt for u64 work.